### PR TITLE
Correct frida docs

### PR DESCRIPTION
**Pull request by nsurbay**
Check with https://qbdi.readthedocs.io/en/docs-frida/frida_bindings.html

### CODE REVIEW

File: docs/source/frida_bindings.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Frida bindings
 ==============
 
-We propose bindings for Frida, the whole C/C++ API is available through them, but they are also backed up by helpers that fluidify script writing.
+We propose bindings for Frida, the whole C/C++ API is available through them, but they are also backed up by helpers that fluidifies script writing.
```
Comment (bla5r) [inline docs/source/frida_bindings.rst:4]: I think 'helpers that fluidify' was fine.

**Review by bla5r — CHANGES_REQUESTED** [review]
Apart from the typo, LGTM.

File: docs/source/frida_bindings.rst (hunk 1 above)
Comment (0xAcid) [reply docs/source/frida_bindings.rst:4]: Indeed, my bad.

File: docs/source/frida_bindings.rst [hunk 2]
```diff
@@ -67,6 +67,98 @@ Analysis
     :members: QBDI.prototype.getInstAnalysis, QBDI.prototype.getInstMemoryAccess, QBDI.prototype.getBBMemoryAccess
     :member-order: alphabetical
 
+.. js:class:: InstAnalysis
+
+  Object describes the Analysis of the current instruction
```
Comment (romainthomas) [inline docs/source/frida_bindings.rst:72]: **This** object  or Object that describes ?

File: docs/source/frida_bindings.rst [hunk 3]
```diff
@@ -67,6 +67,98 @@ Analysis
...
+    Structure containing analysis results of an operand provided by the VM (if ANALYSIS_OPERANDS)
+
+  .. js:attribute:: symbol
+
+    Instruction symbol (if ANALYSIS_SYMBOL and found)
+
+
+.. js:class:: MemoryAccess
+
+  Object describes a memory access.
```
Comment (romainthomas) [inline docs/source/frida_bindings.rst:137]: Object that describes ?

Comment (0xAcid) [reply docs/source/frida_bindings.rst:137]: It was originalyly `Object described a memory access.` that I changed to `[this] Object describes a memory access.` ("this" is implied here, but we could also used your formulation.

Comment (nsurbay) [reply docs/source/frida_bindings.rst]: fix

File: docs/source/frida_bindings.rst (hunk 2 above)
Comment (nsurbay) [reply docs/source/frida_bindings.rst]: fix

**Pull request merged**